### PR TITLE
Update and rename Rambler.xml to Rambler.ru.xml

### PR DESCRIPTION
**Pull request by Folant**

### CODE REVIEW

**Comment by J0WI** [comment]
Some of the previous test urls are no longer redirected by the new rules. Can you explain why you dropped them?

**Comment by Folant** [comment]
- cnt.rambler.ru 403, blocked by uBlock
- cntcerber.rambler.ru - 404
- counter.rambler.ru - 403, uBlock
- api.news.rambler.ru - 404
- scounter.rambler.ru - 403, uBlock
- sync.rambler.ru - 204 [No content], uBlock
- topline.rambler.ru - 403
- *-realty.rambler.ru urls redirected to news.rambler.ru/realty
- Also missed m.kassa.rambler.ru and update.rambler.ru.
Should I add some (all?) of these as targets?

**Comment by J0WI** [comment]
Yes, please.
Not every HTTPS Everywhere user also uses uBlock. You can try to find some valid urls on search engines or by browsing on the site, that we have a 200 response for test urls.

File: src/chrome/content/rules/Rambler.ru.xml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+	<target host="vladikavkaz-realty.rambler.ru" />
+	<target host="vladimir-realty.rambler.ru" />
+	<target host="yoshkar-ola-realty.rambler.ru" />	
+	<target host="rcounter.rambler.ru" />
+	<test url="http://rcounter.rambler.ru/rcounter/rcounter.min.js" />
+	<test url="http://rcounter.rambler.ru/rcounter/rcounter.topline.min.js" />
+	<test url="http://rcounter.rambler.ru/rcounter/clickmap.min.js" />
+	<target host="reklama.rambler.ru" />
+	<!--<target host="scounter.rambler.ru" />
+	<test url="http://scounter.rambler.ru/top100.scn?4402382" /> -->
```
Comment (J0WI) [inline src/chrome/content/rules/Rambler.ru.xml]: Can you document the error with the host that are commented out?

**Pull request merged**